File: Javascript/ejercicio23.js
let n;
let numero;
let i;

n = prompt("Ingrese un número entero positivo");
n = parseInt(n);

if (isNaN(n) || n <= 0) {
    console.log("¡No ingresaste un número válido!");
    alert("¡No ingresaste un número válido!");
} else {
    let Suma = 0;
    for (let i = 1; i <= n; i += 2) {
        Suma += i;
    }//FinPara
    console.log("La suma de los números enteros impares menores o iguales a " + n + " es:  " + Suma);
    alert("La suma de los números enteros impares menores o iguales a " + n + " es:  " + Suma);
}//FinAlgoritmo

File: Phython/practica/ejercicio23.py


entrada = input("Ingrese un número entero positivo")

try:

    n = int(entrada)
    print("¡No ingresaste un número válido!")

    if n < 0:
        print("¡No ingresaste un número positivo!")

    else:
        Suma = 0
        for i in range(1, n + 1):
            Suma += i
    #FinPara
    print("La suma de los números enteros impares menores o iguales a " , n , " es:  " + Suma);
    
except ValueError:
    print("¡No ingresaste un número válido!")
#FinAlgoritmo